libs/ledger-live-common/src/exchange/providers/sell.ts

```diff
@@ -18,44 +18,36 @@ const testSellProvider: ExchangeProviderNameAndSignature = {
   version: 2,
 };
 
-let providerDataCache: Record<string, ExchangeProviderNameAndSignature> | null = null;
-
 /**
  * The result is cached after the first successful fetch to avoid redundant network calls.
```

Comment:
We can remove this comment, since you remove the cache
```suggestion
```

Reply:
I have remove the complete comment. Thanks!